saleor/graphql/core/mutations.py

```diff
@@ -173,6 +174,7 @@ def clean_instance(cls, instance, errors):
                 if field in cls._meta.exclude:
                     continue
                 for message in message_dict[field]:
+                    field = snake_to_camel_case(field)
```

Comment:
Shouldn't we call `snak_to_camel_case` inside `add_error` instead? We would then ensure that every error you add to the response (as you can do it outside of `clean_instance`) is correctly formatted.